README.md

```diff
@@ -1,13 +1,76 @@
-# pygame-Coin-Fall-
+# pygame-Coin-Fall
+---
+>Collect as many coins from the coin fall in 30 seconds. Avoid the bombs!
 
 [![Join the chat at https://gitter.im/pygame-Coin-Fall/Lobby](https://badges.gitter.im/pygame-Coin-Fall/Lobby.svg)](https://gitter.im/pygame-Coin-Fall/Lobby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-Made using pygame.
-Works on Python 2.x and pygame
+[![forthebadge made-with-python](http://ForTheBadge.com/images/badges/made-with-python.svg)](https://www.python.org/) 
+
+
+
+## Table of content
+
+- [Introduction](#introduction)
+  - [Technologies](#technologies)
+  - [Working](#how-it-works)
+- [Installation guide](#installation-guide)
+- [Screenshot](#screenshot)
+- [License](#license)
+
+
+## Introduction
+---
+[(Back to top)](#table-of-content)
+
+A strategy board game made with python. More like a shorter version of chess, where one needs to stay focused, anticipate the other player's moves and cover all the tracks.
+
+### Technologies:
+- `Python 2.x`
+- `pygame` [tutorial](https://pythonprogramming.net/pygame-python-3-part-1-intro/)
```

Comment:
We don't need external tutorial links, if something needs to be explained, let us do it. It is our README afterall.